docs/release-notes/2.22.0.md

```diff
@@ -0,0 +1,44 @@
+# Percona Monitoring and Management 2.22.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">September 21, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## New Features
+
+- [PMM-8396](https://jira.percona.com/browse/PMM-8396): Backup Management: Retry failed scheduled backups
+- [PMM-8079](https://jira.percona.com/browse/PMM-8079): DBaaS: Upgrade Operators inside Kubernetes cluster
+- [PMM-7924](https://jira.percona.com/browse/PMM-7924): MySQL Performance Details dashboard: Add "Performance Schema Status Monitoring" chart
+
+## Improvements
+
+- [PMM-8688](https://jira.percona.com/browse/PMM-8688): Backup Management: Action button "Create Backup" mislabeled
+- [PMM-8311](https://jira.percona.com/browse/PMM-8311): Integrated Alerting: Disable edit/delete buttons for Percona-sourced Templates
+- [PMM-8077](https://jira.percona.com/browse/PMM-8077): DBaaS: Show the current version of Operators used in Kubernetes
+- [PMM-8509](https://jira.percona.com/browse/PMM-8509): "Check for updates" can be disabled in Configuration->Settings->Advanced settings
+- [PMM-7392](https://jira.percona.com/browse/PMM-7392): DBaaS: Change Number of Nodes when editing Topology
+
+## Bugs Fixed
+
+- [PMM-8613](https://jira.percona.com/browse/PMM-8613): Backup Management: Unable to restore MySQL backup
+- [PMM-8463](https://jira.percona.com/browse/PMM-8463): Backup Management: State stuck on "Pending" when creating backup with already existing name
+- [PMM-8408](https://jira.percona.com/browse/PMM-8408): DBaaS: `pmm-client` Docker image incorrectly tagged with development version
```

Comment:
```suggestion
- [PMM-8408](https://jira.percona.com/browse/PMM-8408): DBaaS: Wrong (development) version of PMM Client used for monitoring DB Clusters created by DBaaS
```

Reply:
- [PMM-8408](https://jira.percona.com/browse/PMM-8408): DBaaS: Development version of PMM Client was used for monitoring DB Clusters created by DBaaS